src/main/java/com/develhope/imsdevelhope/app/business/services/CustomUserDetails.java

```diff
@@ -0,0 +1,55 @@
+package com.develhope.imsdevelhope.app.business.services;
+
+import java.util.Collection;
+import com.develhope.imsdevelhope.app.persistence.entities.User;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+public class CustomUserDetails implements UserDetails {
```

Comment:
mark it as a service with `@Service` so that it can be located by the other beans.